.vscode/settings.json

```diff
@@ -22,7 +22,7 @@
     "*.html.erb": "erb"
   },
   "[ruby]": {
-    "editor.defaultFormatter": "Shopify.ruby-lsp"
+    "editor.defaultFormatter": "testdouble.vscode-standard-ruby"
```

Comment:
i dont know if it was only here, but the ruby lsp was not formating while saving, as expected on this vscode settings. so i changed to use the standard it just resolved.

if this was specifically in my environment I can change back, no problem